Breadcrumbs.py

```diff
@@ -1,94 +1,115 @@
 # -*- coding: utf-8 -*-
-import sublime, sublime_plugin
-import math
+import re
+import html
```

Comment:
The commands for phantoms and popups (both in their current form are only compatible with recent st3 builds) are already unavailable in st2 via

```
def is_visible(self):
    return int(sublime.version()) > 3124
```

... which seems to be the canonical way to achieve this. Since st2 doesn’t have minihtml we can simply import html only for st3. If the dependencies ever become more complex we can re-decide then. Also, I expect st2 support to be quite temporary. I wouldn’t be surprised to see st3 go stable before the end of the year.